pkg/cmd/root.go

```diff
@@ -47,8 +47,14 @@ func init() {
 	RootCmd.PersistentFlags().BoolVar(&disableDockerIntegration,
 		"disable-docker-integration", false, "Do not attempt to list/stop/start"+
 			" docker containers which are using dvol volumes")
+}
+
+func initConfig() {
+	if _, err := os.Stat(basePath); err != nil {
+		os.MkdirAll(basePath, 0600)
+	}
 
 	viper.SetConfigName("config")
-	viper.AddConfigPath("$HOME/.dvol")
+	viper.AddConfigPath(basePath)
 	viper.ReadInConfig()
```

Comment:
The error that it reports is that the config file cannot be read as it does not exist initially.

I think the alternative would be to report the error if the file exists and there is an error, or maybe create it by default on first run. What are your thoughts?